test/usecase/simplestUsecase.js

```diff
@@ -420,6 +428,83 @@ describe('A use case', () => {
             assert.ok(ret.isErr)
             assert.deepStrictEqual(ret.err, { request: [{ notDefined: true }] })
         })
+
+        describe('as an entity', () => {
```

Comment:
there is already a section on this file `should audit` (line 331). all the scenarios about audit should be there.

Reply:
I think the wat I put gives us a better reading of the test context.

Look:

This way:
"A use case" + "the simplest use case with request" + **"as an entity"** + "should audit" + "tests"

The way you mentioned:
"A use case" + "the simplest use case with request" + "should audit" + "tests"

This way we create a section to encourage others to test requests as entities in the future.

What do you think?